### PR TITLE
Features/health screen UI

### PR DESCRIPTION
**Pull request by lucaszcai**

### CODE REVIEW

File: lib/screens/health/vaccine_history/add_custom_vaccine_screen.dart [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+                                      color: Color(0xff575b5f),
+                                      //StyleConstants.darkPurpleGrey,
+                                      width: 2.0),
+                                ),
+                                enabledBorder: OutlineInputBorder(
+                                  borderSide: BorderSide(
+                                    //color: StyleConstants.darkPurpleGrey,
+                                      color: Color(0xff575b5f),
+                                      width: 1.5),
+                                ),
```
Comment (YuanSamuel) [inline lib/screens/health/vaccine_history/add_custom_vaccine_screen.dart:58]: Maybe you could extract these to be inside StyleConstants

File: lib/screens/health/vaccine_history/add_custom_vaccine_screen.dart [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+                                hintText: 'Vaccine Name',
+                                hintStyle: TextStyle(
+                                  fontSize: 15.0,
+                                  fontWeight: FontWeight.w600,
+                                  //color: StyleConstants.darkPurpleGrey)),
+                                  color: Color(0xffbfbfbf),
```
Comment (YuanSamuel) [inline lib/screens/health/vaccine_history/add_custom_vaccine_screen.dart:64]: Could we put this inside StyleConstants?

File: lib/screens/health/vaccine_history/add_vaccine_screen.dart [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+              SizedBox(
+                height: height * 0.02,
+              ),
+              Divider(),
+              SizedBox(
+                height: height * 0.02,
+              ),
+              GestureDetector(
+                onTap: () {
+                  Navigator.push(context, MaterialPageRoute(builder: (_) => AddCustomVaccine()));
```
Comment (YuanSamuel) [inline lib/screens/health/vaccine_history/add_vaccine_screen.dart:47]: Use auto_router

File: lib/screens/health/vaccine_history/vaccine_widget.dart [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+import 'package:petcode_app/utils/style_constants.dart';
+import 'package:provider/provider.dart';
+
+class VaccineWidget extends StatefulWidget {
+
+
+  @override
+  _VaccineWidgetState createState() => _VaccineWidgetState();
+}
```
Comment (YuanSamuel) [inline lib/screens/health/vaccine_history/vaccine_widget.dart:11]: Try to use Stateless if possible

**Review by YuanSamuel — CHANGES_REQUESTED** [review]

File: lib/screens/health/health_screen.dart [hunk 5]
```diff
@@ -0,0 +1,380 @@
...
+          child: SingleChildScrollView(
+            physics: ClampingScrollPhysics(),
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.start,
+              children: [
+                SizedBox(
+                  height: height * 0.05,
+                ),
+                GestureDetector(
+                  onTap: () {Navigator.push(context, MaterialPageRoute(builder: (_) => RemindersScreen()));},
```
Comment (YuanSamuel) [inline lib/screens/health/health_screen.dart]: can you use autorouter lol

**Review by YuanSamuel — CHANGES_REQUESTED** [review]

File: lib/screens/health/health_screen.dart (hunk 5 above)
Comment (lucaszcai) [reply lib/screens/health/health_screen.dart]: I did, that code is outdated I think

**Review by YuanSamuel — APPROVED** [review]

**Pull request merged**